sharktank/sharktank/ops/signatures.py

```diff
@@ -1008,6 +1011,30 @@ def _module_register_buffer_trampoline(
         d.fail(args)
 
 
+@overridable
+def quantize(
+    tensor: AnyTensor, quantizer: AnyTensor, name: str = UnnamedTensorName
```

Comment:
quantizer's type signature should be a QuantizerTensor, no?

Reply:
It can be a sharded tensor containing quantizer tensors.